lib/igniter/code/common.ex

```diff
@@ -766,22 +777,10 @@ defmodule Igniter.Code.Common do
     end
   end
 
-  @doc "Moves the zipper right n times, returning `:error` if it can't move that many times."
+  @deprecated "Use `move_right/2` instead, passing an integer as the second argument."
   @spec nth_right(Zipper.t(), non_neg_integer()) :: {:ok, Zipper.t()} | :error
```

Comment:
So, we don't *have* to do this, but this is a good case to add an updater that will update this in other apps automatically. We can use `Igniter.Refactor.Rename` to do this, since the new function is a superset of the existing function.

Reply:
See this: https://github.com/ash-project/igniter/blob/main/lib/mix/tasks/igniter.upgrade_igniter.ex

Reply:
Sounds good! Will add that and implement `move_left` as discussed in #111.